libs/barista-components/filter-field/src/filter-field-config.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { InjectionToken } from '@angular/core';
+import { defaultTagDataForFilterValuesParser } from './filter-field-util';
+import { DtFilterValue, DtFilterFieldTagData } from './types';
+
+/** User defined configurations that can override some settings of the filter-field component */
+export interface DtFilterValuesConfig {
+  tagValuesParser: (
+    filterValues: DtFilterValue[],
+    editable?: boolean,
+    deletable?: boolean,
+  ) => DtFilterFieldTagData | null;
+}
+
+/** Injection token for the external configuration of the filter-field component */
+export const DT_FILTER_VALUES_CONFIG = new InjectionToken<DtFilterValuesConfig>(
+  'dt-filter-value-config',
+);
+
+export const DT_FILTER_VALUES_DEFAULT_CONFIG: DtFilterValuesConfig = {
+  tagValuesParser: defaultTagDataForFilterValuesParser,
+};
```

Comment:
Maybe we can create an injection token for the tagValuesParser function itself. We try to stay away from what can potentially be a larger configuration object. If we add an injection token for each of the configs, we can control this more granular.